services/core/shared/core/transactionStatistics.js

```diff
@@ -14,11 +14,17 @@
  *
  */
 const moment = require('moment');
+const BigNumber = require('big-number');
 
 const config = require('../../config');
-const transactionStatisticsQueue = require('./transactionStatisticsQueue');
+const requestAll = require('../requestAll');
+const { getTransactions } = require('./transactions');
+const { initializeQueue } = require('./queue');
 const getDbInstance = require('../pouchdb');
 
+const queueName = 'transactionStatisticsQueue';
+const queueOptions = config.queue[queueName] || config.queue.defaults;
```

Comment:
Default values should automatically be picked up if specific configs are not defined. Please check the suggested changes in `services/core/shared/core/queue.js`.
```suggestion
const queueOptions = config.queue[queueName];
```

Reply:
Done!